stories/topics/proc-azure-cancel-deployment.adoc

```diff
@@ -0,0 +1,33 @@
+[id="azure-cancel-deployment_{context}"]
+
+= Canceling {AAPonAzureName} deployments
+
+[role="_abstract"]
+You can gracefully cancel an {AAPonAzureName} deployment. 
+
+.Procedure
+
+. Login to the deployment engine to display the progress of the deployment steps in the *Ansible Automation Platform Deployment Engine* page.
+Refer to xref:azure-monitor-deployment-engine_aap-azure-install[Monitoring deployments on the {PlatformNameShort} Deployment Engine] for information on accessing and logging into the *Ansible Automation Platform Deployment Engine* page.
+
+. To cancel the deployment, click btn:[Cancel Deployment] and confirm.
++
+This action cancels all the remaining steps in the deployment, including the currently running step. It also cancels pending steps in the deployment.
++
+The status for the steps that have not been executed updates to *Cancelled*.
+To view the deployment processes on Azure, open the *Overview* page for your deployment on the Azure web interface and select *Deployments*.
```

Comment:
I think this needs to be changed to "To view the deployment processes on Azure, open the "Overview page" for your managed resource group in which you deployed the Ansible Automation Platform on the Azure web interface and select "Deployments".